superflore/generators/ebuild/ebuild.py

```diff
@@ -250,9 +244,10 @@ def get_ebuild_text(self, distributor, license_text, die_msg=None):
         ret += "\"\n\n"
 
         # SLOT
-        ret += "SLOT=\"{}\"\n".format(self.distro)
+        ret += "SLOT=\"0\"\n"
```

Comment:
Yes. In an email from @aballier, he recommended that I have different `USE` flags for each distro
(e.g. `ros_distro_lunar`), then have different slots for the versions of the packages. The next step will be to populate the `SLOT` field correctly.